pages/design-tokens/opacity.md

```diff
@@ -105,21 +105,6 @@ Your context and coding style determine how you access USWDS opacity tokens in c
           </span>
         </td>
       </tr>
-      <tr>
-        <th scope="row" data-title="Context">
-          <span class="font-lang-3">setting</span>
-        </th>
-        <td data-title="Description">
-          <span>
-            <a href="{{ site.baseurl }}/design-tokens/opacity/" class="token">opacity</a>
-          </span>
-        </td>
-        <td data-title="Example">
-          <span>
-            $theme-hover-opacity: <code>50</code>
-          </span>
-        </td>
-      </tr>
```

Comment:
> [!Note]
> I removed some settings examples in this PR because they do not exist in USWDS and I couldn't find an alternative theme setting that accepted these tokens. I was hoping to reduce confusion so that users didn't try to bring a non-existent setting into their config. Please flag if you feel that removing these examples is too opinionated or would cause more confusion!